src/runtime_src/core/tools/xbutil2/SubCmdStatus.cpp

```diff
@@ -0,0 +1,605 @@
+/**
```

Comment:
Please schedule a meeting with me regarding the need for this new command "status".

This is looking to be more of a report "via examine" then its own separate command.

Reply:
Sure. I will schedule a meeting.